src/pages/Graph/SummaryPanelGraph.tsx

```diff
@@ -275,7 +275,7 @@ export default class SummaryPanelGraph extends React.Component<SummaryPanelPropT
               NS
             </Badge>
           </Tooltip>
-          {ns}
+          {ns}{' '}
```

Comment:
Doesn't this add space in all cases?  This is minor and I guess ir doesn't really matter but this way there still may be inconsistent space between the namespace name and the validation status. No big deal, can leave it.

Reply:
Yes, it does but I think it is irrelevant given it keeps the codes base simple and the human eye can barely discriminate the extra 5 pixels:

![Kiali_Console](https://user-images.githubusercontent.com/1312165/73996323-307ca180-4910-11ea-9df0-df11b3ccecc5.png)

Reply:
I'm totally willing to remedy this if anyone feels it warrants the extra 'if' clause.

Reply:
Here is an image without the extra 5px spacing:
![Kiali_Console](https://user-images.githubusercontent.com/1312165/73996888-d11f9100-4911-11ea-94bd-7c3285d76f1b.png)

Hardly discernible to the human eye.

But again, I'm open to whatever ppl think.

Reply:
It make sense. And it keeps the code cleaner, I guess.